crates/holochain/src/conductor/config.rs

```diff
@@ -195,11 +200,45 @@ pub mod tests {
                     init_params: "some_params".into()
                 }),
                 passphrase_service: Some(PassphraseServiceConfig::Cmd),
+                keystore_path: None,
                 admin_interfaces: Some(vec![AdminInterfaceConfig {
                     driver: InterfaceDriver::Websocket { port: 1234 }
                 }]),
                 use_dangerous_test_keystore: true,
             }
         );
     }
+
+    #[test]
+    fn test_config_keystore() {
+        let toml = r#"
+    environment_path = "/path/to/env"
+    use_dangerous_test_keystore = true
```

Comment:
Just wondering about this? Is the point of this PR to still be using the test keystore?

Reply:
Might be off topic for this  PR though

Reply:
For the unit tests we want to use the test keystore - holoport configs will not set this to true